DashAI/back/tasks/translation_task.py

```diff
@@ -3,30 +3,19 @@
 from datasets import Dataset
 from datasets.dataset_dict import DatasetDict
 
-from DashAI.back.tasks.task import Task
+from DashAI.back.tasks.base_task import BaseTask
 
 
-class TranslationTask(Task):
+class TranslationTask(BaseTask):
     """
     Abstract class for translation tasks.
     Here you can change the methods provided by class TranslationTask.
     """
 
-    NAME: str = "TranslationTask"
+    name: str = "TranslationTask"
     SOURCE: str = ""
     TARGET: str = ""
 
-    @staticmethod
-    def create():
-        task = TranslationTask()
-        return task
-
-    def validate_dataset(self, dataset: DatasetDict, class_column: Union[str, int]):
-        """
-        TODO: Implement this validation
-        """
-        return None
```

Comment:
Why remove this? Didn't we agree that tasks validate their datasets? Even though this method is not yet implemented